demos/device_defender_for_aws/demo_config.h

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
This needs to be renamed to "defender_demo_config.h" and placed into each vendor's config_file folder.

Reply:
How about merge the content of this with into "defender_config.h " and then rename as "aws_defender_config.h"? This will follow the same naming rule that used by shadow demo. (Shadow demo has the config file named as  "aws_shadow_config.h")

Reply:
The new redesigned device shadow demo use shadow_config.h, the old shadow CSDK v4 beta uses aws_shadow_config.h.
To follow the other demos please call it "defender_demo_config.h"

Reply:
I see.